resources/static/js/Analytics.js

```diff
@@ -0,0 +1,409 @@
+                    var myChart = new Chart(ctx, {
+                        type: 'bar',
+                        data: {
+                            labels: ['Temp 1', 'Temp 2', 'Temp 3'],
+                            datasets: [{
+                                label: 'Monto total Online',
+                               data: [OnlineTempTotalOne, OnlineTempTotalTwo, OnlineTempTotalThree ],
+                                backgroundColor: [
+                                    'rgba(38, 235, 14, 0.7)',
+                                    'rgba(38, 235, 14, 0.7)',
+                                    'rgba(38, 235, 14, 0.7)',
+                                ],
+                                borderColor: [
+                                    'rgba(38, 235, 14, 1)',
+                                    'rgba(38, 235, 14, 1)',
+                                    'rgba(38, 235, 14, 1)',
+                                ],
+                                borderWidth: 2
+                            }]
+                        },
+                        options: {
+                        	legend: {
+                                labels: {
+                                    fontColor: 'white'
+                                }
+                        	},
+                            scales: {
+                                yAxes: [{
+                                    ticks: {
+                                        beginAtZero: true,
+                                        fontColor: 'white'
+                                    }
+                                }],
+                                xAxes: [{
+                                    ticks: {
+                                        fontColor: 'white'
+                                    },
+                                }]
+                            }
+                        }
+                    });
+                    
+                    var ctx = document.getElementById('myChart6').getContext('2d');
+                    var myChart = new Chart(ctx, {
+                        type: 'line',
+                        data: {
+                            labels: ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio','Julio'
+                            ,'Agosto','Septiembre','Octubre','Noviembre','Diciembre'],
+                            datasets: [{
+                               label: 'Tarifas promedio',
+                               data: [ContentList[0].averageRate, ContentList[1].averageRate, ContentList[2].averageRate,
+                                    ContentList[3].averageRate, ContentList[4].averageRate, ContentList[5].averageRate,
+                                    ContentList[6].averageRate, ContentList[7].averageRate, ContentList[8].averageRate,
+                                    ContentList[9].averageRate, ContentList[10].averageRate, ContentList[11].averageRate],
```

Comment:
Nuevamente el tema de los subíndices y su posible reemplazo por un array, construyendo esto en un for.